chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -115,9 +115,9 @@ void BraveRenderViewContextMenu::AddSpellCheckServiceItem(
 void BraveRenderViewContextMenu::InitMenu() {
   RenderViewContextMenu_Chromium::InitMenu();
 
+  int index = -1;
```

Comment:
This might give an unused local variable warning (or error because I think it's treated as an error) under certain conditions.

Reply:
addressed in 2025088c945de7d44c240623355c7ececb4d6786